automation_tools/repository.py

```diff
@@ -175,6 +174,11 @@ def enable_satellite_repos(cdn=False, beta=False, disable_enabled=True,
         if sat_version == '6.3' and puppet4 == 'yes':
             repos.append('rhel-{0}-server-satellite-{1}-puppet4-rpms')
 
```

Comment:
else: 
      repos.append('rhel-{0}-server-satellite-maintenance-6-rpms')

Reply:
@san7ket I was a little confused with what you meant here, but fixed it to what i think you meant.